utility/start_server.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+python /app/manage.py migrate
+if [ "${BUILD_ENV}" = "development" ] ; then
```

Comment:
I think this section should be removed now that we're telling people to run setup_test.sh. BUILD_ENV is a build-time argument but this script isn't run while the image is built.

Reply:
Would like to discuss with you about this been doing some refactor today and had some things I want to talk about in terms of args defaults and the runtime/architecture of my setup. Also not sure what you mean by setup_test.sh

Reply:
I think it would make more sense to check `if [ "$DJANGO_SETTINGS_MODULE" = "config.settings.local" ];` here instead of BUILD_ENV. Since start_server.sh runs as the container starts up, not at build time.

Earlier you had instructions in CONTRIBUTING.rst which said to run setup_test.sh, so I thought you were trying to switch to that instead of doing the data population here. Never mind...but please don't forget to remove setup_test.sh since it's no longer used.